packages/gatsby/src/commands/develop-process.ts

```diff
@@ -1,7 +1,7 @@
 import { syncStaticDir } from "../utils/get-static-dir"
 import reporter from "gatsby-cli/lib/reporter"
-import chalk from "chalk"
```

Comment:
@sidharthachatterjee Did you mean to remove this? It's breaking builds.

Reply:
@ascorbic I removed it because it was apparently unused 🤷🏼 I'll add it back